generator/src/generator/mod.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) fn generate(module: &xcbgen::defs::Module) -> Vec<Generated> {
 
     let mut main_proto_out = Output::new();
     let mut main_x11rb_out = Output::new();
-    for out in [&mut main_proto_out, &mut main_x11rb_out] {
+    for out in vec![&mut main_proto_out, &mut main_x11rb_out].into_iter() {
```

Comment:
I guess we should leave a comment about this in https://github.com/psychon/x11rb/issues/538.